Ports/file/package.sh

```diff
@@ -6,3 +6,26 @@ use_fresh_config_sub='true'
 files=(
     "http://ftp.astron.com/pub/file/file-${version}.tar.gz#fc97f51029bb0e2c9f4e3bffefdaf678f0e039ee872b9de5c002a6d09c784d82"
 )
+
+configure() {
+    host_env
+    run autoreconf -f -i
```

Comment:
Hmm, that’s what the readme for developers said to do after checking out from the sources, is it a problem because our tarballs are preconfigured and dont really need this?

Reply:
It isn't a problem in itself, but it makes the result depend on the version of autotools (and other libraries) that is installed on the host (which may lead to problems later).